dataland-frontend/src/components/generics/ViewFrameworkBase.vue

```diff
@@ -24,7 +24,24 @@
               @change="handleChangeFrameworkEvent"
             />
             <slot name="reportingPeriodDropdown" />
+            <div class="flex align-content-start align-items-center pl-3">
+              <div
+                v-if="dataType != DataTypeEnum.EutaxonomyNonFinancials && dataType !== DataTypeEnum.Sme"
+                data-test="hideEmptyDataToggle"
+                class="form-field vertical-middle"
+              >
+                <InputSwitch
+                  data-test="dataPointToggleButton"
+                  inputId="dataPointIsAvailableSwitch"
+                  v-model="hideEmptyFields"
+                />
+                <span data-test="hideEmptyDataToggle" class="ml-2 font-semibold" style="font-size: 14px">
+                  {{ "Hide empty fields" }}
```

Comment:
Unneccessary complicated. `{{ "Hide empty fields" }}` could be `Hide empty fields`.

Reply:
Changed

Reply:
Almost ""

Reply:
Gone